tests/pytests/unit/modules/test_pip.py

```diff
@@ -10,6 +10,10 @@
 from salt.exceptions import CommandExecutionError
 from tests.support.mock import MagicMock, patch
 
+TARGET = []
+if os.environ.get('VENV_PIP_TARGET'):
```

Comment:
A new environment variable that affects the test suite should probably be documented.  
Also, if this environment variable *isn't* set do the tests run as they did before?

Reply:
Ah, where do we document env variables?

Note that this variable isn't here to influence the tests; rather, if you define this variable, then that influences pip, (pip automatically adds the `--target` param) and therefore, these tests fail. 

And of course, when the variable isn't set, the tests continue working as before. We should, however, verify this by running the pipeline (I believe you need to approve the run?).